### ISSUE TITLE
Travis failing on new PR's

### ISSUE THREAD

**Issue by ajurna** [OpenPoGo/OpenPoGoBot#258]
### Expected Behavior

Travis should give a green report on new pr's that pass testing.
a previous change has
### Actual Behavior

failing on stepper test
pokemongo_bot/tests/stepper_test.py:215: AssertionError
### Steps to Reproduce

submit a pr
### Other Information

OS: 
Git Commit:

**Comment by andythorne** [comment]
Seem to be running fine on the latest build:

```
$ py.test --cov=pokemongo_bot --cov=plugins pokemongo_bot/ plugins/
============================= test session starts ==============================
platform linux -- Python 3.5.2, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /home/travis/build/OpenPoGo/OpenPoGoBot, inifile: 
plugins: cov-2.3.1
collected 49 items 
pokemongo_bot/tests/event_manager_test.py ........
pokemongo_bot/tests/logger_test.py ..
pokemongo_bot/tests/mapper_test.py ....
pokemongo_bot/tests/plugins_test.py .....
pokemongo_bot/tests/stepper_test.py ..........
pokemongo_bot/tests/navigation/camper_navigator_test.py ....
pokemongo_bot/tests/navigation/destination_test.py ....
pokemongo_bot/tests/navigation/fort_navigator_test.py ..
pokemongo_bot/tests/navigation/waypoint_navigator_test.py ......
pokemongo_bot/tests/navigation/path_finder/direct_path_finder_test.py .
pokemongo_bot/tests/navigation/path_finder/google_path_finder_test.py ...
----------- coverage: platform linux, python 3.5.2-final-0 -----------
Name                                                         Stmts   Miss  Cover
--------------------------------------------------------------------------------
plugins/__init__.py                                              0      0   100%
plugins/catch_pokemon/__init__.py                               97     97     0%
plugins/egg_incubator/__init__.py                               40     40     0%
plugins/evolver/__init__.py                                     70     70     0%
plugins/recycle_items/__init__.py                               36     36     0%
plugins/socket/__init__.py                                      25     25     0%
plugins/socket/botevents.py                                     53     53     0%
plugins/socket/myjson.py                                         5      5     0%
plugins/socket/uievents.py                                      68     68     0%
plugins/spin_pokestop/__init__.py                               93     93     0%
plugins/transfer_pokemon/__init__.py                            93     93     0%
plugins/web/__init__.py                                         93     93     0%
pokemongo_bot/__init__.py                                      229    172    25%
pokemongo_bot/event_manager.py                                  78      0   100%
pokemongo_bot/human_behaviour.py                                 9      0   100%
pokemongo_bot/item_list.py                                      32      0   100%
pokemongo_bot/logger.py                                         28      3    89%
pokemongo_bot/mapper.py                                         41      0   100%
pokemongo_bot/navigation/__init__.py                             3      0   100%
pokemongo_bot/navigation/camper_navigator.py                    25      0   100%
pokemongo_bot/navigation/destination.py                         17      0   100%
pokemongo_bot/navigation/fort_navigator.py                      17      0   100%
pokemongo_bot/navigation/navigator.py                            7      0   100%
pokemongo_bot/navigation/path_finder/__init__.py                 2      0   100%
pokemongo_bot/navigation/path_finder/direct_path_finder.py       4      0   100%
pokemongo_bot/navigation/path_finder/google_path_finder.py      21      0   100%
pokemongo_bot/navigation/path_finder/path_finder.py              4      0   100%
pokemongo_bot/navigation/waypoint_navigator.py                  27      0   100%
pokemongo_bot/plugins.py                                        46      0   100%
pokemongo_bot/stepper.py                                        78      1    99%
pokemongo_bot/utils.py                                          53     26    51%
--------------------------------------------------------------------------------
TOTAL                                                         1394    875    37%
========================== 49 passed in 19.17 seconds ==========================
The command "py.test --cov=pokemongo_bot --cov=plugins pokemongo_bot/ plugins/" exited with 0.
```

**Comment by ajurna** [comment]
if you look at the results from
#257 or #259 . they seem to be not running.
the errors are unrelated to the code changes.

**Comment by andythorne** [comment]
Both are failing because your code is failing the pylint checks:

https://travis-ci.org/OpenPoGo/OpenPoGoBot/jobs/151243120#L1057-L1065
https://travis-ci.org/OpenPoGo/OpenPoGoBot/jobs/151243120#L1057-L1065

Fix these and the build will pass.

**Comment by andythorne** [comment]
Oh i see you make a commit to fix the tests in those PRs.

I think the issue is from #254. It detects we are in 30 meters and exits before we have take a step.

For some reason #254 was merged before the build finished.

**Comment by andythorne** [comment]
Will be fixed in #261
